### PR TITLE
Ahoy! Hacktober fest is in the corneeeeer :)

### PR DESCRIPTION
**Pull request by xeviknal**
New hacktober new complains :)

### CODE REVIEW

**Comment by xeviknal** [comment]
Review:

1. Go to `files changed` tab
2. Click to `Review changes` button
3. express what do you think about the change.
Comment type: If you want to leave a message but don't want to approve the change. Suggestions mainly.
Approve type: changes look good to you and you want these changes accepted.
Require changes: there is something wrong on the code. Ask the owner to change something.
4. If you believe the change can be accepted, press the green button right below "Merge pull request".

File: README.md [hunk 1]
```diff
@@ -1,21 +1,2 @@
...
-a tricky way to make designers feel as IT experts. :D
-
-## Xevi, I guess you know how it works, right?
-sure. still stuck with sth wierd
-
-bailongo, stop living in holidays, please. 
-
-## complains area
-
-### Cinnamon complains
```
Comment (xeviknal) [inline README.md:15]: Good old times when @ramyroquay was obsessed with cinnamon... what are you addicted to now?

Comment (piscastudio) [reply README.md:15]: To Don Patricio!

File: README.md [hunk 2]
```diff
@@ -1,21 +1,2 @@
...
-## Irma, what do you think about CSS?
-a tricky way to make designers feel as IT experts. :D
-
-## Xevi, I guess you know how it works, right?
-sure. still stuck with sth wierd
-
-bailongo, stop living in holidays, please. 
-
-## complains area
-
```
Comment (piscastudio) [inline README.md:14]: This is fucking awkward >.<'

File: README.md [hunk 3]
```diff
@@ -1,21 +1,2 @@
...
-### Cinnamon complains
-No puc dir res en contra de la canyella, em sap greu :(
-M'encanta el te de canyella amb extra de canyella
-El que no m'agrada és que em posa massa hot
-
-### Xevi complains
-- M'agradaria queixarme de les vacances del xevi, em semblen ofensives.
+# Hacktoberfest 2019 for Pixar Studio S.A
+Who starts? Who fancy to complain about the team mates?
```
Comment (piscastudio) [inline README.md:2]: don't know how to use it xD

**Pull request merged**